packages/libs/react-ui/src/components/Modal/Modal.stories.tsx

```diff
@@ -6,11 +6,20 @@ import React from 'react';
 
 const meta: Meta<{ title?: string } & IModalProps> = {
   title: 'Layout/Modal',
+  parameters: {
+    docs: {
+      description: {
+        component:
+          'Component which informs user about a task and it may sometimes contain important information, require decision making, or involve tasks for the user.',
```

Comment:
The description description should inform the user on how this component can be used and the component involved rather than giving an explanation of what modals do generally. Essentially we want to make it easier for people to use the components. Looking at the contents of this modal component, I think it needs to be cleaned up so I would say leave this description blank for now